embulk-input-jdbc/src/main/java/org/embulk/input/jdbc/AbstractJdbcInputPlugin.java

```diff
@@ -185,6 +185,7 @@ public ConfigDiff transaction(ConfigSource config,
         Schema schema;
         try (JdbcInputConnection con = newConnection(task)) {
             // TODO incremental_columns is not set => get primary key
+            con.showDriverVersion();
```

Comment:
The comment above (`// TODO incremental_columns ...`) should be above the following line (`schema = setupTask(con, task);`).